modules/store/src/utils.ts

```diff
@@ -92,10 +93,16 @@ export function createReducerFactory<T, V extends Action = Action>(
   metaReducers?: MetaReducer<T, V>[]
 ): ActionReducerFactory<T, V> {
   if (Array.isArray(metaReducers) && metaReducers.length > 0) {
-    return compose.apply(null, [...metaReducers, reducerFactory]);
+    reducerFactory = compose.apply(null, [...metaReducers, reducerFactory]);
   }
 
-  return reducerFactory;
+  return (reducers: ActionReducerMap<T, V>, initialState?: InitialState<T>) => {
+    const reducer = reducerFactory(reducers, initialState);
```

Comment:
Passing the `initialState` to the factory brings some confusion for me 😅 

We can remove it because we're already providing an initial state via the following code, making it obsolete.
```
state = state === undefined ? (initialState as T) : state;
```

Besides this, it LGTM!


PS: Sorry for doing the review partially.

Reply:
Thanks!
I see that, you'r right.
It just breaks one of the tests... so I will remove that and update the failing test :)